content/wayneconnolly/drone-rancher-stack-v1/index.md

```diff
@@ -0,0 +1,58 @@
+---
+date: 2018-07-02T00:00:00+00:00
+title: rancher-stack-v1
```

Comment:
The title should be something human readable like Rancher Stack v1.

Reply:
I have updated as per your suggestion. Re: secrets: I don't provide secrets. If you want to do that then Ill have to update the image. I'm happy to as secrets should be used. Actually - gimme a day to do that. I'm at work now so can't test.

Reply:
No problem, but you should always allow secrets for credentials ;)